### PR TITLE
Updating packer documentation to make usage easier to find

### PR DESCRIPTION
**Pull request by cboneti**
### Submission Checklist

* [x] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [x] Are all tests passing? (`make tests`)
* [ ] Have you written unit tests to cover this change?
* [x] Is unit test coverage still above 80%?
* [x] Have you updated all applicable documentation?
* [x] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

File: modules/packer/custom-image/README.md [hunk 1]
```diff
@@ -30,6 +30,29 @@ images to internal projects.
 [hpcimage]: https://cloud.google.com/compute/docs/instances/create-hpc-vm
 [startup-metadata]: https://cloud.google.com/compute/docs/instances/startup-scripts/linux
 
+## Example blueprints
+
+Several examples blueprints that create a startup-scripts using the HPC Toolkit [startup-script] module and then use it to build a specialized image can be found below.
```
Comment (nick-stroud) [inline modules/packer/custom-image/README.md:35]: nit: This sentence feels long and hard to follow. I would suggest:

> A recommended pattern for building images with this module is to use the terraform based [startup-script] module along with this packer `custom-image` module. Below you can find links to several examples of this pattern, including usage instructions.

File: modules/packer/custom-image/README.md [hunk 2]
```diff
@@ -30,6 +30,29 @@ images to internal projects.
...
+
+### Intel-Optimized Slurm Cluster
+
+The [Intel-Optimized] Slurm Cluster [blueprint](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/community/examples/intel/hpc-cluster-intel-select.yaml)
+Intel Select Solutions within the Toolkit. It adds the intel compliance software on top of a Slurm on GCP image.
+
+[Image Builder]: https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/examples/image-builder.yaml
+[startup-script]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/modules/scripts/startup-script
+[examples README]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/examples#image-builderyaml-
+[Intel-Optimized]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/examples/intel#intel-optimized-slurm-cluster
```
Comment (nick-stroud) [inline modules/packer/custom-image/README.md:54]: It is recommended to use relative paths for all of these as it will preserve the branch being viewed. All these links point to main, but if you use relative path then if I am browsing from the develop branch the link will keep me on the develop branch.

File: modules/packer/custom-image/README.md [hunk 3]
```diff
@@ -30,6 +30,29 @@ images to internal projects.
...
+* The [HPC VM Image][hpcimage] as a base upon which to customize
+* A VPC network with firewall rules that allow IAP-based SSH tunnels
+* A Toolkit runner that installs a custom script
+
+Please review the [examples README] for usage instructions.
+
+### Intel-Optimized Slurm Cluster
+
+The [Intel-Optimized] Slurm Cluster [blueprint](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/community/examples/intel/hpc-cluster-intel-select.yaml)
+Intel Select Solutions within the Toolkit. It adds the intel compliance software on top of a Slurm on GCP image.
```
Comment (nick-stroud) [inline modules/packer/custom-image/README.md:49]: nit: s/intel/Intel

File: modules/packer/custom-image/README.md [hunk 4]
```diff
@@ -30,6 +30,29 @@ images to internal projects.
...
+
+* The [HPC VM Image][hpcimage] as a base upon which to customize
+* A VPC network with firewall rules that allow IAP-based SSH tunnels
+* A Toolkit runner that installs a custom script
+
+Please review the [examples README] for usage instructions.
+
+### Intel-Optimized Slurm Cluster
+
+The [Intel-Optimized] Slurm Cluster [blueprint](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/community/examples/intel/hpc-cluster-intel-select.yaml)
```
Comment (nick-stroud) [inline modules/packer/custom-image/README.md:48]: nit: This sentence feels like something is missing. It reads: `The Intel-Optimized Slurm Cluster blueprint Intel Select Solutions within the Toolkit`. 

Maybe instead: `The Intel-Optimized Slurm Cluster blueprint adds Intel compliance software on top of a Slurm on GCP image.` and leave it at that?

File: modules/packer/custom-image/README.md (hunk 1 above)
Comment (cboneti) [reply modules/packer/custom-image/README.md:35]: PTAL

File: modules/packer/custom-image/README.md (hunk 2 above)
Comment (cboneti) [reply modules/packer/custom-image/README.md:54]: oh, I did not know that, thanks!

Comment (cboneti) [reply modules/packer/custom-image/README.md:54]: changed

**Review by nick-stroud — APPROVED** [review]
